include/openthread/channel_manager.h

```diff
@@ -47,8 +47,12 @@ extern "C" {
  * @brief
  *   This module includes functions for Channel Manager.
  *
- *   The functions in this module are available when Channel Manager feature
- *   (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_ENABLE`) is enabled. Channel Manager is available only on an FTD build.
+ *   The functions in this module are available when Channel Manager features
+ *   (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_ENABLE`) and (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE`)
+ *   are enabled. Channel Manager behavior depends on the device role. It manages the network-wide Pan channel on a Full
+ *   Thread Device in rx-on-when-idle mode, or with (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE`) set,
+ *   selects Csl channel in synchronized rx-off-when-idle mode. On a Minimal Thread Device
+ *   (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE`) selects the Csl channel.
```

Comment:
Can you use PAN and CSL throughout the comments, rather than Pan/pan and Csl/csl?